src/components/views/elements/ReplyThread.js

```diff
@@ -185,8 +189,20 @@ export default class ReplyThread extends React.Component {
 
     componentWillUnmount() {
         this.unmounted = true;
+        if (this.room) {
+            this.room.removeListener("Room.redaction", this.onRoomRedaction);
+            this.room.removeListener("Room.redactionCancelled", this.onRoomRedaction);
+        }
     }
 
+    onRoomRedaction = (ev, room) => {
+        if (this.unmounted) return;
+
+        // we could skip an update if the event isn't in our timeline,
```

Comment:
we should do this as I imagine large accounts are going to be stuck in render loops with this change.